daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/Root.scala

```diff
@@ -0,0 +1,20 @@
+package edu.illinois.ncsa.daffodil.dsom
+
+import edu.illinois.ncsa.daffodil.grammar.RootGrammarMixin
+import edu.illinois.ncsa.daffodil.xml.NamedQName
+
+final class Root(parentArg: SchemaDocument,
```

Comment:
Scaladoc. The Root is a specialized kind of ElementRef that doesn't live within a model group.